### DISCUSSION TITLE
Feedback on cockle repro capstone graph please!! :)

### DISCUSSION THREAD

**Discussion by drlawson** [RobertsLab/resources#1134] [Q&A] [answered]
Hi all, not sure if I am doing this whole GitHub thing correctly but I was trying to put together a graph based on what we discussed in lab meeting. How does this look? Is there too much information in one graph? Thanks for all your help everyone!!

For the second image I was thinking something like that based on the "panel" idea we were discussing... but much less blurry. Let me know if you think that seems like a good plan or if I should just keep it separate :)
<img width="511" alt="image" src="https://user-images.githubusercontent.com/73162507/110087808-fbc45f00-7d48-11eb-9629-451be8a86555.png">
<img width="609" alt="Screen Shot 2021-03-05 at 12 34 40 AM" src="https://user-images.githubusercontent.com/73162507/110089135-94a7aa00-7d4a-11eb-9454-fdff9b6527dc.png">

**Comment by sr320** [comment, marked as answer]
Looks better - some feedback - https://d.pr/v/seBa10


Biggest thing is looking at graphs there is no indication of fact these are hermaphroditic.
Each organism needs to be represented somehow.  Can you provide a link to the data?
